sentry_sdk/tracing_utils.py

```diff
@@ -132,6 +134,8 @@ def record_sql_queries(
         data["db.paramstyle"] = paramstyle
     if executemany:
         data["db.executemany"] = True
+    if record_cursor_repr and cursor is not None:
+        data["db.cursor"] = cursor
```

Comment:
I decided to add this to the tracing utils, to allow recording the cursor instance. 

I can see this as being useful if someone does subsequent fetches from a cursor.